src/platforms/ContractCall/ContractCallReward.tsx

```diff
@@ -61,7 +67,14 @@ const ContractCallReward = ({
         </>
       }
     >
-      <AvailabilityTags rolePlatform={platform} />
+      {contractCallFunction === ContractCallFunction.DEPRECATED_SIMPLE_CLAIM ? (
```

Comment:
Maybe we could abstract this condition into NftAvailabilityTags, so we wouldn't need to repeat it here and useContractCallCardProps? Just use NftAvailabilityTags as it'll remain, so the legacy support in it is just an implementation detail that we'll be able to remove later

Reply:
Good point, refactored this in 2a773621dd6dfbb5b1cfff124104b60404b35d26